roadmap.asciidoc

```diff
@@ -24,33 +24,26 @@ As our engineering capacity is limited, we're focusing our efforts on a single r
 i.e. patch releases for the current stable minor release (e.g. 1.2.1.Final) are done in case of critical bug fixes only.
 ====
 
-== 2.1 - December 2022
-
-* Add new SMT to generate diff change events (exposes only changed fields)
-* Embedded Engine improvements
-* Oracle connector performance improvements
-* Add new storage module for S3
-* Debezium UI - Support editing existing connector configurations
-* Debezium UI - Support visualization of single message transform (SMT) payloads
-* Debezium UI - Support pass-thru connector configuration parameters
-
 == 2.2 - March 2023
 
 * Support configurable signaling channels
-* Add schema history compaction tool
 * Add new JDBC Sink Connector that natively supports Debezium events
-* Rewrite Cassandra connector to use the common connector framework (run within Debezium Server)
+* New Infinispan Sink connector for Debezium Server
+* New RocketSMQ Sink connector for Debezium Server
+* Add new SMT to generate diff change events (exposes only changed fields)
+* Rewrite Cassandra connector to use the common connector framework (run within Debezium Server) (tentative)
```

Comment:
This is already done.

Reply:
I saw the ISPN sink connector this morning.  I think its worth keeping under 2.2 until that's out the door in March, wouldn't you agree?

Reply:
@Naros  Yes, ISPN is fine. Just Cassandra work is completed.